README.md

```diff
@@ -331,4 +344,11 @@ The `outbound-edi` function can be invoked via the UI for testing.
 
   </details>
 
-5. You can view the file using the [Buckets Web View](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
\ No newline at end of file
+5. You can view the file using the [Buckets Web View](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
+
+# Customizing the Bootstrap
+
+The bootstrap uses a sample trading partner to set up and test the read and write EDI workflows. You can customize the bootstrap for your use case by doing one or all of the following:
+- [Add a partner profile](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#add-a-trading-partner-profile) to replace the test trading partner with your real trading partners' details and requirements.
+- [Create Stedi Mappings](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages) that transform EDI documents into a custom JSON shape for your internal system. The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. 
```

Comment:
```suggestion
- [Create a Stedi mapping](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages) that transform EDI documents into a custom JSON shape for your internal system. The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. 
```

Reply:
_Stedi Mappings_ is the product. We don’t want our customers to create that; we’d have nothing left to do! ;-)